### ISSUE TITLE
Cannot build the game in my machine

### ISSUE THREAD

**Issue by aviskarkc10** [squgeim/minesweeper#2]
I am on Ubuntu and running `make build` gives me the following issues:
```
src/libminesweeper/minesweeper.c: In function ‘msw_is_game_over’:
src/libminesweeper/minesweeper.c:207:5: error: unknown type name ‘uint8_t’
     uint8_t i, j;
     ^
```
This I could work around by including `stdint.h` in the file.

Then I get the error saying no build file is present for which I have to create a directory `bin`

Also when I run the game. All I get is a blank screen and when I terminate it I get the following output waiting for me:
```
minesweeper[8596]: Initializing game with size: 32x16
minesweeper[8596]: Initializing game with size: 32x16
minesweeper[8596]: Total bombs: 76
```
 I have created a PR #1 for the first two

**Comment by kabirbaidhya** [comment]
I think this is specific to the compiler used. @squgeim Can we also document the gcc version we used to build it? I need it for working on [this](https://github.com/squgeim/minesweeper/issues/3) as well.

**Comment by squgeim** [comment]
@kabirbaidhya I have included my GCC config in the readme now.

Fixed by #1
